### DISCUSSION TITLE
What properties control the text size and padding of context menus?

### DISCUSSION THREAD

**Discussion by e-t-l** [nptr/msstyleEditor#151] [Q&A] [answered]
When applying a custom theme (specifically [Rectify Dark](https://github.com/Rectify11/Installer/blob/legacy-installer/Resources/Themes/themes/Rectified/Dark.msstyles)), I notice that the text size and padding of right-click context menus ~and mouseover/hover tooltips~ are increased. I would like to edit the theme to make it the same as default Windows. In the screenshots below, you can see the default Windows Dark Theme followed by the Rectify11 Dark theme, where the R11 elements are slightly larger. ~(curiously, for the tooltips, only padding is increased while text size is actually the same).~

Could anyone help direct me to what properties I should edit in the R11 msstyles to make these elements match the values in the default Windows msstyles?

EDIT: Actually I think I found the properties for tooltips. The class is just called Tooltip. Still looking for context menu and would appreciate help!

Context menu: 

![image](https://github.com/user-attachments/assets/3729f459-e0a5-4431-904b-3df748029f5c)

![image](https://github.com/user-attachments/assets/b4bb5496-76af-4a3a-be72-8b129f7a078e)

~Tooltip:~ 

![image](https://github.com/user-attachments/assets/5e1cf2f7-0f73-4112-b29e-c041824e38b1)

![image](https://github.com/user-attachments/assets/cf0f58e7-9284-4e3b-bc79-4d5f1be924b4)

**Comment by Undisputed00x** [comment, marked as answer]
RClick Menu: `DarkMode::ImmersiveStart::Menu` -> Part27 
Context Menus on Taskbar tray and dropdown menus: `Menu` + `ImmersiveStart::Menu` -> Part27

**Reply by e-t-l** [reply]
ah, so it's called "Part27". Thank you, I never would have figured that out by myself!

**Reply by e-t-l** [reply]
By the way, do you know if nptr prefers that questions like this one be opened as a "Help Request" issue? My intuition for discussing uses of a program are to keep it in the Discussions section, while Issues should be for bugs or feature requests in the program itself, but maybe this repo prefers to keep it all as Issues?
